controllers/video.go

```diff
@@ -194,14 +206,20 @@ func GetVideo(w http.ResponseWriter, r *http.Request, db *sql.DB) {
 		WHERE
 			video_id=?
 	`)
+
 	if err != nil {
-		log.Fatal(err)
+		log.Println(err)
+		http.Error(w, "Error Retreiving Record", http.StatusInternalServerError)
 	}
+
 	defer detailsQuery.Close()
+
 	var title, description string
 	err = detailsQuery.QueryRow(video_id).Scan(&title, &description)
 	if err != nil {
-		log.Fatal(err)
+		log.Println(err)
+		http.Error(w, "Error Retreiving Record", http.StatusInternalServerError)
+		return
```

Comment:
What happens when no rows are returned?
In case of no rows, return a 404 not found.